### ISSUE TITLE
Preferences: Configure no-File-Browser mode

### ISSUE THREAD

**Issue by ghost** [Beep6581/RawTherapee#6692]
I am working only on gigapixel images, and i really like how powerful RT is! 💯 

But...

If you start RT, it shows this screen:

![image](https://user-images.githubusercontent.com/126161988/221687293-d1c793a6-c705-4a9c-b8a3-dacf9071a664.png)

If i would not clean it for the image it has a long loading time because of my big images.

But...

**My working experience is only to use context menu and "open with RawTherapee"**. Now:

- _RT first loads the file manager_
- _than it opens a new window with the image editor_

I want to skip this <ins>allways</ins> via an option in the settings like **"allways skip file manager when opening from context menu"** to reduce the loading time by all of the not needed file manager screen

**Comment by ghost** [comment]
btw that would mean that the double click with the mouse can be reassigned and assigned to the zoom for 0 and 100%...

**Comment by Lawrence37** [comment]
I thought the default behavior already opens just the editor. See remote mode: http://rawpedia.rawtherapee.com/Command-Line_Options#RawTherapee_GUI

**Comment by bediicco** [comment]
Oh, how great! I didn't expect something like this to hide behind a -R! Great 👌🏻👍🏻

**Comment by bediicco** [comment]
@Lawrence37 for me this is fully ok to permanently set -R but i saw now you changed the title and added a label

**Comment by bediicco** [comment]
@Lawrence37 

Where do I write -R (on Windows)? I have no idea where to quickly change this for the mouse context menu.

I believe we need a setting to enable?

My english is not as good, everywhere it sounds like -R is default, but it isn`t. I tried at home and at business. Windows 10 Pro 64 opens first the file manager than the editor.

I can make a video :)

**Comment by ghost** [comment]
_I deleted my comments and summarized them here for brevity:_

RT 5.9 on Windows _does not have the editor view as default_. It first opens the file-manager and than the editor window.

**How do i use** `-R` **option?**

If i use it (_one time_) via CMD

```batch
C:\Program Files\RawTherapee\5.9>rawtherapee -R
```
every new CMD window session now starts directly the editor window when opening an image like

```batch
C:\Program Files\RawTherapee\5.9>rawtherapee test.jpg
```

**But, no effect via mouse context menu** _"open with RawTherapee"_

Therefore: i just don't know how to assign `-R` to the context menu functionality.

**Comment by ghost** [comment]
@Lawrence37 

Oh, there is a difference in handling:

![image](https://user-images.githubusercontent.com/126161988/221966897-2e92b119-8302-4c4b-b315-9e7d15752da6.png)

![image](https://user-images.githubusercontent.com/126161988/221968165-1d4fc658-6d61-4c76-93e4-aabe52bcccc7.png)

I never used the second entry of "open with" :-)

**Comment by Lawrence37** [comment]
-R does the opposite of what you want. Right now, the "Open with RawTherapee" option uses -R (see #3933). The work-around is to use Windows's own "Open with" as you have discovered.

**Comment by ghost** [comment]
@Lawrence37 

Ok, but this is a fresh new windows installation on my home PC. I have never used an -R option because i cannot edit your items in context menu on windows.

The only test i did was at my business laptop -R in the CMD console window.

The differences and settings are up to your installer. Not me.

**Comment by Lawrence37** [comment]
Maybe there is a language barrier. Let me rephrase.

If you choose "Open with RawTherapee", it will automatically use -R. It was designed that way. If you do not want the file browser to show, do not use "Open with RawTherapee". Use the "Open with" (Öffnen mit) instead. That one does not use the -R.

**Comment by ghost** [comment]
@Lawrence37 

Ok, yes. I didn't understand (read) it was designed this way (to have different kind of "open with...")

Ok, for me this solves my issue if this is fixed and will not change because of error correction in some later releases.

Thank you :)
